mso/resource_mso_schema_site_bd.go

```diff
@@ -71,22 +77,25 @@ func resourceMSOSchemaSiteBdImport(d *schema.ResourceData, m interface{}) ([]*sc
 	if err != nil {
 		return nil, fmt.Errorf("No Sites found")
 	}
-	stateSite := get_attribute[2]
+	site_template := regexp.MustCompile("-").Split(get_attribute[2], 2)
+	stateSite := site_template[0]
+	stateTemplate := site_template[1]
 	found := false
 	statebd := get_attribute[4]
-	for i := 0; i < count; i++ {
+	for i := 0; i < count && !found; i++ {
 		tempCont, err := cont.ArrayElement(i, "sites")
 		if err != nil {
 			return nil, err
 		}
 		apiSite := models.StripQuotes(tempCont.S("siteId").String())
+		apiTemplate := models.StripQuotes(tempCont.S("templateName").String())
 
-		if apiSite == stateSite {
+		if apiSite == stateSite && apiTemplate == stateTemplate {
 			bdCount, err := tempCont.ArrayCount("bds")
 			if err != nil {
 				return nil, fmt.Errorf("Unable to get bd list")
 			}
-			for j := 0; j < bdCount; j++ {
+			for j := 0; j < bdCount && !found; j++ {
```

Comment:
Do we need this !found here? Is the break after found not already breaking the for loop?